draft-ietf-quic-tls.md

```diff
@@ -2170,10 +2188,10 @@ packet = 4cfe4189655e5cd55c41f69080575d7999c25a5bfb
 
 # Analysis of Limits on AEAD_AES_128_CCM Usage {#ccm-bounds}
 
-TLS {{?TLS13}} and {{AEBounds}} do not specify limits on usage for
-AEAD_AES_128_CCM. However, any AEAD that is used with QUIC requires limits on
-use that ensure that both confidentiality and integrity are preserved. This
-section documents that analysis.
+TLS {{?TLS13}} and {{AEBounds}} do not specify per-key or per-connection
+limits on usage for AEAD_AES_128_CCM. However, any AEAD that is used with QUIC
+requires limits on use that ensure that both confidentiality and integrity are
+preserved. This section documents that analysis.
```

Comment:
Perhaps change to: "This section documents an analysis of per-key limits, which - absent an analysis of multi-user security - also set per-connection limits."